kolibri/core/auth/test/test_datasets.py

```diff
@@ -71,3 +71,9 @@ def test_dataset_representation(self):
         )
         new_dataset = FacilityDataset.objects.create()
         self.assertEqual(str(new_dataset), "FacilityDataset (no associated Facility)")
+
+    def test_dataset_incompatible_setting(self):
+        with self.assertRaises(IntegrityError):
```

Comment:
You have one failing test, as this needs to be updated to the `IncompatibleDeviceSettingError` now!